src/bp-templates/bp-nouveau/includes/template-tags.php

```diff
@@ -2608,15 +2608,17 @@ function bp_nouveau_signup_terms_privacy() {
 			<h1><?php echo esc_html( get_the_title( $terms ) ); ?></h1>
 			<?php
 			$get_terms = get_post( $terms );
-			echo apply_filters( 'the_content', $get_terms->post_content );
+			$get_term_content = bp_strip_script_and_style_tags( apply_filters( 'the_content', $get_terms->post_content ) );
+			echo apply_filters( 'bp_term_of_service_content', wp_kses_post( $get_term_content ), $get_terms->post_content );
```

Comment:
```suggestion
			echo apply_filters( 'bp_term_of_service_content', wp_kses_post( bp_strip_script_and_style_tags( apply_filters( 'the_content', $get_terms->post_content ) ) ), $get_terms->post_content );
```
And remove 2611 line